db/migrate/20221019132624_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] I was unable to set up your app locally because this migration already exists. Kindly remove this file to fix that.